client/modules/i18n/startup.js

```diff
@@ -9,6 +9,7 @@ import { Reaction } from "/client/api";
 import { Shops, Translations } from "/lib/collections";
 import * as Schemas from "/lib/collections/schemas";
 import i18next, { packageNamespaces, getLabelsFor, getMessagesFor, i18nextDep } from "./main";
+import {mergeDeep} from "/lib/api";
```

Comment:
Looks like we need to add the `space-in-brackets` rule (and make arrays an exception).

http://eslint.org/docs/rules/space-in-brackets

Probably something like...

```js
"space-in-brackets": [2, "always", {
  "objectsInArrays": false,
  "arraysInArrays": false,
  "arraysInObjects": false
}]
```